sample_custom_component2/main.go

```diff
@@ -0,0 +1,222 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"github.com/Azure/azure-storage-fuse/v2/common/config"
+	"github.com/Azure/azure-storage-fuse/v2/common/log"
+	"github.com/Azure/azure-storage-fuse/v2/external"
+	"github.com/Azure/azure-storage-fuse/v2/internal"
+)
+
+// SAMPLE EXTERNAL COMPONENT IMPLEMENTATION
```

Comment:
Add a comment explaining how to build this component inform of a dll/so file which can be later linked to main code.

Reply:
Sample code shall be clear enough to tell a user on how to compile this code.